phpBB/styles/prosilver/theme/forms.css

```diff
@@ -274,14 +275,19 @@ fieldset.submit-buttons input {
 	min-width: 100%;
 	max-width: 100%;
 	height: 270px;
+	padding-right: 20px;
 	resize: vertical;
+	-webkit-transition: all 0.5s ease;
+	-moz-transition: all 0.5s ease;
+	-ms-transition: all 0.5s ease;
```

Comment:
run everything through autoprefixer as you have way too many prefixes for the proper browser support we use `> 1%, last 2 versions`